src/Components/Search/SearchBar.tsx

```diff
@@ -376,18 +331,15 @@ export class SearchBar extends Component<Props, State> {
     )
   }
 
-  renderInputComponent = props => <SearchInputContainer {...props} />
-
-  renderAutosuggestComponent(t, { xs }) {
-    const { term } = this.state
-    const { viewer } = this.props
+  const renderInputComponent = props => <SearchInputContainer {...props} />
```

Comment:
I think we can pass `SearchInputContainer` component for `renderInputComponent` prop. Like this

```javascript
<Autosuggest
  ...
  renderInputComponent={SearchInputContainer}
/>
```